inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/render/RenderSpan.java

```diff
@@ -44,13 +44,15 @@ public RenderSpan(Offset aOffset)
         end = aOffset.getEnd();
     }
 
-    public RenderSpan(VSpan aVSpan, Span aSpan)
+    public RenderSpan(VSpan aVSpan, Span aSpan, Offset offset)
```

Comment:
Why pass an offset with begin/end if you only ever use the begin? I'd suggest to only pass the begin to make the code easier to understand and maintain.